adminpage/sport/admin/studentAdmin.py

```diff
@@ -114,6 +167,10 @@ def has_enrolled(self, obj):
 
     has_enrolled.boolean = True
 
+    def write_to_telegram(self, obj):
+        return None if obj.telegram is None else \
+            format_html("<a href=\"https://t.me/{}\">{}</a>", obj.telegram[1:], obj.telegram)
```

Comment:
Мне кажется, лучше это открывать в новой вкладке, чтобы вкладка с админкой не терялась
```suggestion
            format_html(
                '<a target="_blank" href="https://t.me/{}">{}</a>',
                obj.telegram[1:],
                obj.telegram
            )
```